File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n, row, column, space;
// prompt user for height
    do
    {
        n = get_int("Height: ");
    }
//height defined
    while (n < 1 || n > 8);
//row
    for (row = 0; row < n; row++)
    {
//space
        for (space = 0; space < n - row - 1; space++)
        {
            printf(" ");
        }
//column
        for (column = 0; column <= row; column++)
        {
            printf("#");
        }
// move to next column
        printf("\n");
    }

}

File: week 6/mario.py
# TODO

from cs50 import get_int
# prompt user for height
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(0, n, 1):
    # print space
    for j in range(0, n, 1):
        if (i+j < n-1):
            print(" ", end="")
        # print #
        else:
            print("#", end="")
    print()